thirdparty/glad/gl.c

```diff
@@ -2627,10 +2614,9 @@ static GLADapiproc glad_dlsym_handle(void* handle, const char *name) {
   typedef __eglMustCastToProperFunctionPointerType (GLAD_API_PTR *PFNEGLGETPROCADDRESSPROC)(const char *name);
 #endif
   extern __eglMustCastToProperFunctionPointerType emscripten_GetProcAddress(const char *name);
-#elif EGL_STATIC
-  typedef void (*__eglMustCastToProperFunctionPointerType)(void);
+#elif defined(GLAD_GLES2_USE_SYSTEM_EGL)
```

Comment:
Most likely, it will need EGL_STATIC branch back to work with ANGLE, the rest of the changes seems unrelated (but I have not tested):

```c++
#elif defined(EGL_STATIC)
  typedef void (*__eglMustCastToProperFunctionPointerType)(void);
  typedef __eglMustCastToProperFunctionPointerType (GLAD_API_PTR *PFNEGLGETPROCADDRESSPROC)(const char *name);
  extern __eglMustCastToProperFunctionPointerType GLAD_API_PTR eglGetProcAddress(const char *name);
#elif defined(GLAD_GLES2_USE_SYSTEM_EGL)